### PR TITLE
Add job definitions to DB seed script

### PR DESCRIPTION
**Pull request by dlqqq**
Also fixes bug encountered where jobs cannot be filtered by job definition ID.

### CODE REVIEW

File: dev/seed.py [hunk 1]
```diff
@@ -17,7 +19,39 @@ def get_db_path():
...
+            "lame definition",
+        ]
+    )
+    job_definition_id = str(uuid4())
+    input_uri = "".join(name.split()) + ".ipynb"
+    output_prefix = ""
+    timezone = random.choice(list(zoneinfo.available_timezones()))
+    # random schedules can be generated via https://crontab.guru/
+    schedule = random.choice(
+        ["0 0,12 1 */2 *", "0 4 8-14 * *", "0 0 1,15 * 3", "@weekly", "5 0 * 8 *", "15 14 1 * *"]
```
Comment (3coins) [inline dev/seed.py:38]: Is "@weekly" a failure case?

File: dev/seed.py [hunk 2]
```diff
@@ -67,29 +102,43 @@ def load_data(count: int, db_path: str):
...
+@click.option("--jobs-count", default=57, help="No of jobs to create, default is 57.")
+@click.option(
+    "--job-defs-count", default=27, help="No of job definitions to create, default is 27."
+)
 @click.option(
     "--db_path",
     type=click.Path(),
     default=get_db_path(),
     help="DB file path, default is scheduler db path",
 )
```
Comment (3coins) [inline dev/seed.py:139]: Can you check if there are shortcut options (e.g., -j -jd -db) for inputs and add them?

File: jupyter_scheduler/scheduler.py [hunk 3]
```diff
@@ -191,7 +191,7 @@ def list_jobs(self, query: ListJobsQuery) -> ListJobsResponse:
             if query.status:
                 jobs = jobs.filter(Job.status == query.status)
             if query.job_definition_id:
-                jobs = jobs.filter(Job.status == query.job_definition_id)
+                jobs = jobs.filter(Job.job_definition_id == query.job_definition_id)
```
Comment (3coins) [inline jupyter_scheduler/scheduler.py:194]: Thanks for this.

**Review by 3coins — COMMENTED** [review]
Looks good. Some minor comments.

File: dev/seed.py (hunk 1 above)
Comment (dlqqq) [reply dev/seed.py:38]: It was generated by crontab.guru, but it looks like it's non-standard. I'll remove it from the schedule stubs.

File: dev/seed.py (hunk 2 above)
Comment (dlqqq) [reply dev/seed.py:139]: Doing this is easy just by adding an additional positional argument to `@click.option()`. One small change though, we will be using double-hyphen (`--j`, `--jd`, `--db`) to specify these CLI arguments require an additional CLI argument after them, to conform to existing conventions.

https://serverfault.com/questions/387935/whats-the-difference-betwen-the-single-dash-and-double-dash-flags-on-shell-comm

**Review by 3coins — APPROVED** [review]

**Pull request merged**